controllers/wavefront_controller_test.go

```diff
@@ -637,14 +594,12 @@ func TestReconcileProxy(t *testing.T) {
 
 	t.Run("can create proxy with histogram ports enabled", func(t *testing.T) {
 		stubKM := testhelper.NewMockKubernetesManager()
-
-		wfSpec := defaultWFSpec()
-		wfSpec.DataExport.WavefrontProxy.Histogram.Port = 40000
-		wfSpec.DataExport.WavefrontProxy.Histogram.MinutePort = 40001
-		wfSpec.DataExport.WavefrontProxy.Histogram.HourPort = 40002
-		wfSpec.DataExport.WavefrontProxy.Histogram.DayPort = 40003
-
-		r, _, _, _ := setupForCreate(wfSpec)
+		r, _, _, _ := setupScenario(wftest.CR(func(w *wf.Wavefront) {
```

Comment:
Minor point, but since we are changing the code, should we change this to be consistent with Jaeger test, that is construct and new Histogram struct vs setting the values individually